pkg/cloud/alibaba/alibaba_test.go

```diff
@@ -0,0 +1 @@
+package alibaba
```

Comment:
Please add a test similar to https://github.com/openshift/cluster-cloud-controller-manager-operator/blob/master/pkg/cloud/aws/aws_test.go

Reply:
@gujingit bump

Reply:
ok, I will add it.